src/commands/generate/types.ts

```diff
@@ -230,6 +242,24 @@ export const ${constantName} = '${hashedKey}' as const`
     private encryptKey(variable: Variable) {
         return `dvc_obfs_${createHash('sha256').update(variable._id).digest('hex')}`
     }
+
+    private updateAliasesInRepo() {
```

Comment:
shouldn't this method only be called if `updateAliases` is true?

Reply:
ya